### ISSUE TITLE
Adjust aria roles on the front page

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7417]
The inner text area could be `search`. The content in the black box could be `main`. The rest below the black area could be `complementary`.
